app/routes/__init__.py

```diff
@@ -1,8 +1,16 @@
-from flask import Blueprint, Flask
+from flask import Flask
 
-bp_api = Blueprint("bp_api", __name__)
```

Comment:
I believe this approach is more suitable if amidst the project we want to add a prefix to the whole api, such as `/api/users`.

Reply:
corrected